### DISCUSSION TITLE
My suggestions for the Oxygen Tank/Welding Fuel changes

### DISCUSSION THREAD

**Discussion by AssaultEngineer** [FakeFishGames/Barotrauma#12619] [Ideas and suggestions]
Im kinda indifferent to the welding fuel part of these changes so far because i feel like for handheld tools that use oxygen/welding fuel it is more of a buff for those as even with bad skills they will still have longer run times and with good skill they might last a very decent amount of time. Maybe they add some new tool variants that can have more tanks/fuel attached or just have better economy/performance compared to normal variants (maybe a welder that uses batteries?). I hope 3 tanks of welding fuel can last as much as maybe 4-5 old tanks with good skill, I don't think i ever used more than that unless it was a very repair heavy mission but i haven't gotten to beat the game yet so i dunno it might need more.

Maybe some suits can have 2 oxygen tanks attached based on their role or something or just all suits can have 2 attached to help make up for the loss of stacking. With an oxygenite tank and regular oxygen tank attached maybe the speed buff will be a little less but you get more dive time because oxygenite tank. More of the medium and high end suits should be able to draw from hull oxygen like i cant imagine ALL of the suits (except PUCS) don't have retractable visors or some other means to allow people to breathe with a helmet on, I mean come on its the future.

i cant think of anything else but im sure other people have better suggestions and if they think my ideas on this are outright bad they can debate me or something.

**Comment by bookslayer10** [comment]
I like the idea of a battery powered welder. Obviously being able to use renewzble battery charge is better than fuel, so it should be harder to use than the standard one. It should repair a bit slower slower than a normal welder, and could have a 3 second warm-up before it starts firing so it takes more skill to use effectively. I think it should be only found as loot, not craftable.

-

Because oxygen tank capacity already got buffed, I don't think a multi-tank dive suit is needed. This update has already doubled the resource efficiency of oxygenite tanks and halved the ammount of times you need to replace your dive suit oxygen canister; it doesn't need to make oxygen any more convenient.

**Reply by Hazmat616** [reply]
Have it have its speed use a skill check so ideally mechanics will be the primary people to use them otherwise it will be too slow.
